vtipnik.py

```diff
@@ -1,5 +1,6 @@
 import sys
-from glob import glob
```

Comment:
Jaká je motivate pro tuto změnu?

Reply:
bylo to z důvodů použití funkce glob.glob na místo glob ve funkci seznam_vtipu

Díky za upozornění změna neměla žádný smysl, opravím.